playwright-e2e/pages/dsm/participantList-page.ts

```diff
@@ -1,14 +1,14 @@
 import { expect, Locator, Page } from '@playwright/test';
 
 import { waitForNoSpinner } from 'utils/test-utils';
-import Checkbox from 'lib/widget/checkbox';
-import ParticipantPage from './participant-page';
-import { Filters } from '../../lib/component/dsm/filters/filters';
+import { Filters } from 'lib/component/dsm/filters/filters';
+import { ParticipantListTable } from 'lib/component/dsm/participantListTable/participantListTable';
```

Comment:
Looks like the folder name were accidentally changed from `table` to `participantListTable`, right?

Reply:
Oh, yes. Fixed - > tables